.github/workflows/callable.writerside-publish.yml

```
@@ -0,0 +1,200 @@
+        run: |
+          git remote rm origin
+          git remote add origin git@github.com:${{ github.repository }}
+          git config --local user.email "actions@github.com"
+          git config --local user.name "GitHub Action"
+
+      - name: Generate javadoc
+        run: ./gradlew javadoc
+
+      - name: Fetch previous version
+        id: perv_version
+        run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)